core/platform/auth/firebase_auth_services_test.py

```
@@ -424,49 +408,54 @@ def revoke_refresh_tokens(self, uid: str) -> None:
         """
-        return self.update_user(uid, custom_claims=custom_claims)
+        self.update_user(uid, custom_claims=custom_claims)
 
     def update_user(
         self,
         uid: str,
         email: Optional[str] = None,
-        disabled: bool = False,
-        custom_claims: Optional[str] = None,
-    ) -> str:
+        disabled: Optional[bool] = None,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)